docs/source/explanation/algorithms/action/action_classification.rst

```diff
@@ -1,14 +1,47 @@
-Action Classificaiton
-=====================
+Action Recognition
+==================
+
+Action recognition is a problem of identifying the action that is being performed in a video. The input to the algorithm is a sequence of video frames, and the output is a label indicating the action that is being performed.
+
+For supervised learning we use the following algorithms components:
+
+- ``Augmentations``: We use standard data augmentations for videos, including random resizing and random cropping, horizontal flipping. We randomly sample a segment of frames from each video during training.
+
+- ``Optimizer``: We use the Adam with weight decay fix (AdamW) optimizer.

```

Comment:
please, add a link. Some users can be not familiar with AdamW or even Adam and it is just a good manner to have a link for components